assets/styl/_components/block-list-with-cards.twig

```diff
@@ -0,0 +1,23 @@
+{#
+ # @extends block.twig
+ #
+ # template for content small block
+ #}
+{% extends 'block-list.twig' %}
+
+{% block block_body_content %}
+    {% for item in list %}
+        <li class="block-list-item" {{ loop.index > itemPerPage ? ' aria-hidden="true"' }}>
+            {% include 'card.twig' with {
```

Comment:
Can be `{{ include('card.twig', {`

Reply:
include function in twig js don't work with third parametter :sad: